downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -117,6 +120,7 @@ object Git : GitBase() {
 
             // To safe network bandwidth, first try to only fetch exactly the revision we want.
             try {
+                log.info { "Trying to fetch only revision '$revision'." }
                 runGitCommand(targetDir, "fetch", "origin", revision)
```

Comment:
Why not also use `--depth` here?

Reply:
Good point. I somehow got misled and thought asking to fetch a revision fetches only that revision and none of its parent, which is not the case.